src/main/java/frc/robot/utilities/TunableNumber.java

```diff
@@ -25,21 +36,22 @@ public TunableNumber(
     this.value = defaultValue;
     this.consumer = consumer;
     if (!Robot.isCompetition) {
+      inst = NetworkTableInstance.getDefault();
       numberEntry = tuningTab.add(name, defaultValue).getEntry();
+      numberEntry.close();
+      
+      doubleTopic = inst.getDoubleTopic(name);
```

Comment:
Since we get the topic from elsewhere, the GenericEntry isn't needed anymore.

```suggestion
      inst = NetworkTableInstance.getDefault();
      tuningTab.add(name, defaultValue);
      
      doubleTopic = inst.getDoubleTopic(name);
```